### PR TITLE
US593 to dev - intro/outro/titlescreen script clean up

### PR DESCRIPTION
**Pull request by dnguye99asu**
Clean up for IntroController.cs, OutroController.cs, and TitleScreenManager.cs.

- [x] The code compiles
- [x] The code has been developer-tested
- [x] The script and each function has a description
- [x] The code follows guidelines listed in Quality Control Practices

### CODE REVIEW

File: Psyche/Assets/Scripts/UI/TitleScreenManager.cs
Comment (jmolive8) [inline Psyche/Assets/Scripts/UI/TitleScreenManager.cs:1]: At the top the _using System;_ shouldn't be needed. Also I think the function names should be Pascal Case.

File: Psyche/Assets/Scripts/UI/OutroController.cs [hunk 1]
```diff
@@ -10,83 +10,95 @@
...
+    public Button LinkButton;
+
+    //variables
+    private Image Img1, Img2, Img3, Img4, ButtonImg, LinkImg;
+    private TMP_Text ButtonText;
     private bool IsTransitioning;
-    private Color buttonColor;
-    int curScreen = 1;
+    private Color ButtonColor;
+    int CurScreen = 1;
```
Comment (jmolive8) [inline Psyche/Assets/Scripts/UI/OutroController.cs:30]: I think this should have _private_ on the front.

File: Psyche/Assets/Scripts/UI/IntroController.cs [hunk 2]
```diff
@@ -8,11 +8,20 @@
...
+    public GameObject Intro4;
+    public GameObject Intro5;
+    public GameObject Intro6;
+    public GameObject Intro7;
+
+    //variables
+    private Image Img1, Img2, Img3, Img4, Img5, Img6, Img7;
     private bool IsTransitioning;
-    int curScreen = 1;
+    int CurScreen = 1;
```
Comment (jmolive8) [inline Psyche/Assets/Scripts/UI/IntroController.cs:24]: _private_ here too.

**Review by jmolive8 — APPROVED** [review]

File: Psyche/Assets/Scripts/UI/OutroController.cs (hunk 1 above)
Comment (dnguye99asu) [reply Psyche/Assets/Scripts/UI/OutroController.cs]: Corrected in commit [here](https://github.com/MissionToPsyche/tungsten_12a_web_game-se/pull/253/commits/9e0100f194fd60ed48e6f56ae8c83340000e2dde)

File: Psyche/Assets/Scripts/UI/IntroController.cs (hunk 2 above)
Comment (dnguye99asu) [reply Psyche/Assets/Scripts/UI/IntroController.cs]: Corrected in commit [here](https://github.com/MissionToPsyche/tungsten_12a_web_game-se/pull/253/commits/9e0100f194fd60ed48e6f56ae8c83340000e2dde) as well

**Pull request merged**